apps/quickstart-integrations-monday-code/src/queue-service.js

```diff
@@ -0,0 +1,33 @@
+import { Logger } from "@mondaycom/apps-sdk";
+import { PubSub } from "@google-cloud/pubsub";
+const pubSubClient = new PubSub();
```

Comment:
Some basic questions:

We don't need to authenticate into pubsub because we're running in google cloud, right? 

Does it support local usage too?

Reply:
I migrated it to the SDK so now it's much easier, and no regarding the authentication since we are running inside the GCP